From 89763320ae8dd53630c297b36c586427cd796744 Mon Sep 17 00:00:00 2001
From: Silvio F <silvio.fricke@gmail.com>
Date: Fri, 15 Nov 2013 13:37:07 +0100
Subject: [PATCH 41/41] v4l2_calls: remove some defines which removed from
 kernel 3.10+

We not able to compile gst-plugins-good with newer Kernel as 3.10:

v4l2_calls.c: In function 'gst_v4l2_fill_lists':
v4l2_calls.c:57:26: error: 'V4L2_CID_HCENTER_DEPRECATED' undeclared (first use in this function)
v4l2_calls.c:300:12: note: in expansion of macro 'V4L2_CID_HCENTER'
v4l2_calls.c:57:26: note: each undeclared identifier is reported only once for each function it appears in
v4l2_calls.c:300:12: note: in expansion of macro 'V4L2_CID_HCENTER'
v4l2_calls.c:60:26: error: 'V4L2_CID_VCENTER_DEPRECATED' undeclared (first use in this function)
v4l2_calls.c:301:12: note: in expansion of macro 'V4L2_CID_VCENTER'

This remove this defines, because the functionality is come with the defines
V4L2_CID_PAN_RESET and V4L2_CID_TILT_RESET

Signed-off-by: Silvio F <silvio.fricke@gmail.com>
---
 sys/v4l2/v4l2_calls.c | 10 ----------
 1 file changed, 10 deletions(-)

diff --git a/sys/v4l2/v4l2_calls.c b/sys/v4l2/v4l2_calls.c
index ca18ef1..0b00d96 100644
--- a/sys/v4l2/v4l2_calls.c
+++ b/sys/v4l2/v4l2_calls.c
@@ -50,14 +50,6 @@
 
 #include "gst/gst-i18n-plugin.h"
 
-/* Those are ioctl calls */
-#ifndef V4L2_CID_HCENTER
-#define V4L2_CID_HCENTER V4L2_CID_HCENTER_DEPRECATED
-#endif
-#ifndef V4L2_CID_VCENTER
-#define V4L2_CID_VCENTER V4L2_CID_VCENTER_DEPRECATED
-#endif
-
 GST_DEBUG_CATEGORY_EXTERN (v4l2_debug);
 #define GST_CAT_DEFAULT v4l2_debug
 
@@ -295,8 +287,6 @@ gst_v4l2_fill_lists (GstV4l2Object * v4l2object)
         break;
       case V4L2_CID_HFLIP:
       case V4L2_CID_VFLIP:
-      case V4L2_CID_HCENTER:
-      case V4L2_CID_VCENTER:
 #ifdef V4L2_CID_PAN_RESET
       case V4L2_CID_PAN_RESET:
 #endif
-- 
1.8.4.2

